angular/src/app/home/home.component.ts

```diff
@@ -65,8 +65,6 @@ export class HomeComponent implements OnInit {
         this.confValues = this.appConfig.getConfig();
         this.PDRAPIURL = this.confValues.PDRAPI;
         this.SDPAPIURL = this.confValues.SDPAPI;
-        // this.forensicsURL = this.PDRAPIURL + "mds9911";
```

Comment:
Leave these lines intact; we will uncomment them when we release the forensics records.

Reply:
Ok. Checking in...